# Request
float32 recharge_x
float32 recharge_y
float32 recharge_yaw

---

# Response
int8 result 

